libs/components/pages/src/lib/modules/page/types/page-layout-type.ts

```diff
@@ -0,0 +1,7 @@
+export type SkyPageLayoutType =
+  | 'auto'
+  | 'fit'
```

Comment:
Just for clarity, we would never have a page that "fits" its layout, but uses a "box" layout?

Reply:
`auto` and `fit` are general-purpose layout types, whereas `box`, `list`, `split-view` and `tabs` are specifically to support a handful of page layouts using SKY UX components as their main content. I added some code comments to clarify that a bit.